.gitignore

```diff
@@ -1,11 +1,13 @@
+/archives
 /i386
 /sparc
 
-components/components.mk
-components/depends.mk
+components/*.mk
+components/**/build
```

Comment:
Comment was wrongly placed, I mean "components/**/build" and "components/meta-packages/history/build".